.github/workflows/continuous-integration.yml

```diff
@@ -520,38 +505,151 @@ jobs:
-          TEST_EXECUTIONS_TABLE_NAME: cypress_test_suite_executions
-          TEST_RESULTS_TABLE_NAME: cypress_test_results
-          TEST_REPORTS_FOLDER_NAME: vets-website-cypress-reports
+          TEST_EXECUTIONS_TABLE_NAME: vets_website_contract_test_suite_executions
+          TEST_RESULTS_TABLE_NAME: vets_website_contract_test_results
+          TEST_REPORTS_FOLDER_NAME: vets-website-contract-test-reports
 
-      # env.MOCHAWESOME_REPORT_RESULTS is set and exported during the above step when the mochawesome report is generated.  It contains the output string for the publish step at the end of the job with the numbers from the Mochawesome report.
+          # env.MOCHAWESOME_REPORT_RESULTS is set and exported during the above step when the mochawesome report is generated.  It contains the output string for the publish step at the end of the job with the numbers from the Mochawesome report.
 
-      # --------------------------
-      # | Publish Testing Report |
-      # --------------------------
+      # -------------------------
+      # | Contract Tests Summary |
+      # -------------------------
 
-      - name: Download video artifacts
-        if: ${{ needs.cypress-tests.result == 'failure' }}
-        uses: actions/download-artifact@v2
+      - name: Contract test results
+        if: ${{ always() }}
+        uses: LouisBrunner/checks-action@v1.1.1
         with:
-          name: cypress-video-artifacts
-          path: testing-tools-team-dashboard-data/cypress-reports/videos/${{ env.UUID }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+          name: Contract Tests Summary
+          conclusion: ${{ needs.contract-tests.result }}
+          output: |
+            {"summary":${{ env.MOCHAWESOME_REPORT_RESULTS }}}
+
+  testing-reports-cypress:
+    name: Testing Reports - Cypress E2E Tests
+    runs-on: ubuntu-latest
+    needs: [cypress-tests-prep, cypress-tests]
+    if: ${{ always() && needs.cypress-tests-prep.outputs.num_containers > 0 && (needs.cypress-tests.result == 'success' || needs.cypress-tests.result == 'failure') }}
+    continue-on-error: true
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Notify Slack about Cypress test failures
+      if: ${{ github.ref == 'refs/heads/master' && needs.cypress-tests.result == 'failure' }}
+      uses: ./.github/workflows/slack-notify
+      continue-on-error: true
+      env:
+        SSL_CERT_DIR: /etc/ssl/certs
+      with:
+        attachments: '[{"mrkdwn_in": ["text"], "color": "danger", "text": "<!here> E2E tests in `vets-website` have failed on the `master` branch, run: <https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}|${{github.run_id}}>"}]'
+        channel_id: C026PD47Z19 #gha-test-status
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+    - name: Configure AWS credentials
```

Comment:
Nit, can we add AWS Cred and get va-vsp-bot in section underneat `Upload BigQuery Data`